components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java

```diff
@@ -1427,6 +1428,7 @@ public void addFederatedAuthenticatorConfig(FederatedAuthenticatorConfig authnCo
             prepStmt1.setString(4, authnConfig.getName());
             prepStmt1.setString(5, authnConfig.getDisplayName());
             prepStmt1.setString(6, authnConfig.getDefinedByType().toString());
+            prepStmt1.setString(7, AuthenticationType.IDENTIFICATION.toString());
```

Comment:
better add a comment mentioning federated authenticators are always supposed authenticate an externally managed user and share user identity and attributes. Therefore, by default these are always of 'identification' type

Reply:
Addressed with commit c4009b8301f34d84dcfd97dfe775da67c9fd14d8